.github/workflows/notify_on_broken_link_detected/install_deps.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+git clone https://github.com/osscameroon/broken_link_checker.git blc
+pip install -r blc/requirements.txt
```

Comment:
It could be great to have the blc on pypi officially... but we can create another issue to have tha official release on pypi not the testing one !